src/features/ITGovernance/Admin/GRBReview/index.tsx

```diff
@@ -327,6 +332,12 @@ const GRBReview = ({
               {t('reviewDetails.text')}
             </p>
 
+            <GRBReviewStatusCard
+              grbReviewType={grbReviewType}
+              grbDate={grbDate}
+              grbReviewStatus={GRBReviewStatus.SCHEDULED}
+            />
```

Comment:
The card should only render if the review has started and `grbReviewStartedAt` has a value.